demisto_sdk/commands/common/update_id_set.py

```diff
@@ -1133,12 +1139,46 @@ def get_general_data(path: str, packs: Dict[str, Dict] = None):
     pack = get_pack_name(path)
     marketplaces = get_item_marketplaces(path, item_data=json_data, packs=packs)
 
+    if not id_ and 'marketplacev2' in marketplaces:
+        id_ = f"{pack}-{name}"
+
     data = create_common_entity_data(path=path, name=name, to_version=toversion, from_version=fromversion, pack=pack, marketplaces=marketplaces)
     if brandname:  # for classifiers
         data['name'] = brandname
     return {id_: data}
 
 
+def get_trigger_data(path: str, packs: Dict[str, Dict] = None):
+    json_data = get_json(path)
+    id_ = json_data.get('RULE_ID')
+    name = json_data.get('name', '')
+    fromversion = json_data.get('fromVersion')
+    toversion = json_data.get('toVersion')
+    pack = get_pack_name(path)
+    marketplaces = get_item_marketplaces(path, item_data=json_data, packs=packs)
+
+    data = create_common_entity_data(path=path, name=name, to_version=toversion, from_version=fromversion, pack=pack, marketplaces=marketplaces)
+
+    return {id_: data}
+
+
+def get_general_xsiam_yaml_data(path: str, packs: Dict[str, Dict] = None):
+    yaml_data = get_yaml(path)
+
+    id_ = yaml_data.get('id')
+    name = yaml_data.get('name', '')
+    fromversion = yaml_data.get('fromxsiamversion')
+    toversion = yaml_data.get('toxsiamversion')
```

Comment:
No need to have this field. We can have a normal toversion and fromversion

Reply:
done